sway/desktop/output.c

```diff
@@ -457,19 +457,47 @@ static void handle_request_state(struct wl_listener *listener, void *data) {
 	struct sway_output *output =
 		wl_container_of(listener, output, request_state);
 	const struct wlr_output_event_request_state *event = data;
+	const struct wlr_output_state *state = event->state;
```

Comment:
There are changes here that shouldn't be here. A bad rebase?

Reply:
Yup, a bad rebase, and that's why I asked if I should close this pr.

Reply:
You don't need to close the PR. You can rewrite the commit with `git commit --amend` then `git push --force` to push the updated commit.

Reply:
@Nefsen402 I squashed all my commits into one, but did it told me to pull from upstream and those commits appeared as if I had made them too for some reason, would have using just used `git push -f` solved it?

Reply:
Yes, you need to force push because you rewrote history. By squashing, you basically remove all the commits you squash, and put a new commit in its place with all the changes. Since part of the process involves removing old commits, force or force-with-lease is necessary